web/src/containers/ScheduleSummary.js

```diff
@@ -1,74 +1,57 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { Fragment } from 'react';
 import { connect } from 'react-redux';
-import ReactTable from 'react-table';
 
 import { t } from '../i18n';
 import Waypoint from './ConnectedWaypoint';
 import { Section, Subsection } from '../components/Section';
 import { selectActivitySchedule } from '../reducers/activities.selectors';
 
-const ScheduleSummary = ({ tableData }) => (
+const ScheduleSummary = ({ activities }) => (
   <Waypoint id="schedule-summary">
     <Section isNumbered id="schedule-summary" resource="scheduleSummary">
       <Subsection id="schedule-summary-table" resource="scheduleSummary.main">
-        {tableData.data.length === 0 ? (
+        {activities.length === 0 ? (
           <div className="p1 h6 alert">
             {t('scheduleSummary.noDataMessage')}
           </div>
         ) : (
-          <ReactTable
-            showPagination={false}
-            resizable={false}
-            minRows={0}
-            className="h6 -striped"
-            {...tableData}
-          />
+          <table className="table-cms">
+            <tbody>
+              {activities.map(({ name: activityName, milestones }, i) => (
+                <Fragment key={activityName}>
+                  <tr>
+                    <th colSpan="2" className="ds-u-font-weight--bold">
+                      Activity {i + 1}: {activityName}
+                    </th>
+                  </tr>
+                  {milestones.map(({ end, name: milestoneName, start }) => (
+                    <tr>
+                      <td className="ds-u-padding-left--3 ds-u-border-right--0">
```

Comment:
This is to get rid of the horizontal padding between cells in a row, to match the mockup in #1446